pom.xml

```diff
@@ -85,7 +85,7 @@
 
     <properties>
         <!-- Artifact versions -->
-        <uk.bom.version>4.0.0-SNAPSHOT</uk.bom.version>
+        <uk.bom.version>4.0.0-pr-230-SNAPSHOT</uk.bom.version>
```

Comment:
Ok, checked with @shaunharrisonFR , and this PR won't actually be merged. It's just to test the build across the environments. It can't be merged with the PR build in place.

Reply:
Hey, yep, it's just to test the build, I will change the version and ask for a new review. Thanks a lot!